Makefile

```diff
@@ -32,22 +34,11 @@ shrink:
 
 PHONY+= plugins
 plugins: check-mods
-	$(call build,goplugin-aws,cmd/goplugin-aws/main.go)
-	$(call build,goplugin-example,cmd/goplugin-example/main.go)
-	$(call build,goplugin-identity,cmd/goplugin-identity/main.go)
-	$(call build,goplugin-puppetdsl,cmd/goplugin-puppetdsl/main.go)
-	$(call build,goplugin-tf-aws,cmd/goplugin-tf-aws/main.go)
-	$(call build,goplugin-tf-azurerm,cmd/goplugin-tf-azurerm/main.go)
-	$(call build,goplugin-tf-github,cmd/goplugin-tf-github/main.go)
-	$(call build,goplugin-tf-google,cmd/goplugin-tf-google/main.go)
-	$(call build,goplugin-tf-kubernetes,cmd/goplugin-tf-kubernetes/main.go)
-
-puppet-dsl:
```

Comment:
I would like to keep this actually. It saves me a lot of time when iterating changes in the framework using the puppet-dsl plugin.

Reply:
Restored.